HarryPotter/Screens/House/View/HouseMemberCell.swift

```diff
@@ -9,6 +9,9 @@ import UIKit
 
 final class HouseMemberCell: UITableViewCell {
     
+    var houseViewController: HouseViewController?
```

Comment:
mmm I don't think it's a good idea to **hold** the whole ViewController here, in cell. I think you can delete it, if you need to communicate with VC, closure is the way to go :) e.g. `var markedFavourite: (() -> Void)?`. Then VC assigns some closure to this, like: `viewModel.markedFavourite = { ... }`, and the ViewModel tells him something by doing `markedFavourite?()`.